docker/docker-compose.prod.yaml

```diff
@@ -5,38 +5,31 @@ services:
   mongo:
     restart: always
 
-  #main containers
   injective-core:
     restart: always
     logging:
       driver: journald
-    ports:
-      - ${INJ_NETWORK_P2P_PORT}:${INJ_NETWORK_P2P_PORT}
-      - ${INJ_NETWORK_GRPC_PORT}:${INJ_NETWORK_GRPC_PORT}
-      - ${INJ_NETWORK_COSMOS_API_PORT}:${INJ_NETWORK_COSMOS_API_PORT}
-      - ${INJ_NETWORK_COSMOS_GRPC_PORT}:${INJ_NETWORK_COSMOS_GRPC_PORT}
-      - ${INJ_NETWORK_COSMOS_GRPC_WEB_PORT}:${INJ_NETWORK_COSMOS_GRPC_WEB_PORT}
-
-  injective-exchange-process:
+
+  indexer-exchange-import:
+    logging:
+      driver: journald
+
+  indexer-exchange-api:
+    restart: always
+    logging:
+      driver: journald
+
+  indexer-exchange-process:
     restart: always
     logging:
       driver: journald
-    ports:
-      - ${INJ_EXCHANGE_PROCESS_CHRONOS_RPC_PORT}:${INJ_EXCHANGE_PROCESS_CHRONOS_RPC_PORT}
 
-  injective-exchange-api:
+  indexer-chronos-api:
```

Comment:
For this stack, we don't need chronos components, only basis of the indexer stack, which is exchange.
@achilleas-kal can confiorm

Reply:
@nmarcetic we do need chronos otherwise they'll have to use our API for volume data. Since they setup their own stack it's best to set this up too so they don't send so many requests on our API.

Reply:
What we don't need is explorer, so for now we should do exchange + chronos. We'll also add event provider soon.

Reply:
Yea, it makes sense, I was thinking about MM more, who want their node. Most external setups were MM-related, the ones I helped at least. 
Ok so, we are leaving chronos @Ri-go

Reply:
Got it! Thanks guys :)